WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressView.swift

```diff
@@ -0,0 +1,166 @@
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Allows merchant to edit the customer provided address of an order.
+///
+struct EditAddressView: View {
+    var body: some View {
+        GeometryReader { geometry in
+            ScrollView {
+                ListHeaderView(text: Localization.detailsSection, alignment: .left)
+                    .padding(.horizontal, insets: geometry.safeAreaInsets)
+                VStack(spacing: 0) {
+                    TitleAndTextFieldRow(title: Localization.firstNameField,
+                                         placeholder: "",
+                                         text: .constant(""),
+                                         symbol: nil,
+                                         keyboardType: .default)
+                    Divider()
+                        .padding(.leading, Constants.dividerPadding)
+                    TitleAndTextFieldRow(title: Localization.lastNameField,
+                                         placeholder: "",
+                                         text: .constant(""),
+                                         symbol: nil,
+                                         keyboardType: .default)
+                    Divider()
+                        .padding(.leading, Constants.dividerPadding)
+                    TitleAndTextFieldRow(title: Localization.emailField,
+                                         placeholder: "",
+                                         text: .constant(""),
+                                         symbol: nil,
+                                         keyboardType: .emailAddress)
+                    Divider()
+                        .padding(.leading, Constants.dividerPadding)
+                    TitleAndTextFieldRow(title: Localization.phoneField,
+                                         placeholder: "",
+                                         text: .constant(""),
+                                         symbol: nil,
+                                         keyboardType: .phonePad)
+                }
+                .padding(.horizontal, insets: geometry.safeAreaInsets)
+                .background(Color(.systemBackground))
+
+                ListHeaderView(text: Localization.shippingAddressSection, alignment: .left)
+                    .padding(.horizontal, insets: geometry.safeAreaInsets)
+                VStack(spacing: 0) {
+                    Group {
+                        TitleAndTextFieldRow(title: Localization.companyField,
+                                             placeholder: Localization.placeholderOptional,
+                                             text: .constant(""),
+                                             symbol: nil,
+                                             keyboardType: .default)
+                        Divider()
+                            .padding(.leading, insets: geometry.safeAreaInsets)
+                            .padding(.leading, Constants.dividerPadding)
```

Comment:
Why do we need a double padding here? Was it a mistake or is it a weird `SwiftUI` view?

Reply:
I just removed them and it all looks correct!
<img width="383" alt="Screen Shot 2021-09-01 at 3 25 18 PM" src="https://user-images.githubusercontent.com/562080/131740375-c43509f4-a4bc-4b5e-af72-318c2babca2f.png">

Reply:
Looks like I forgot to remove internal padding on second section, while it's correct on first. Thanks for noticing!
Fixed in f25ff68f8fb5841dc2e0734963a5294011460d6a